.github/workflows/tests.yml

```diff
@@ -0,0 +1,56 @@
+name: Run tests
+
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches:
+      - main
+
+jobs:
+    test-php:
+        name: Test on php ${{ matrix.php-version }} and ${{ matrix.os }}
+        runs-on: ${{ matrix.os }}
+        continue-on-error: ${{ matrix.experimental }}
+        strategy:
+            fail-fast: false
+            matrix:
+                experimental: [false]
+                coverage-extension: [pcov]
+                php-version: ["7.1", "7.2", "7.3", "7.4", "8.0"]
+                os: [ubuntu-latest]
+                include:
+                    - { php-version: '5.3', experimental: false, os: ubuntu-latest, coverage-extension: 'xdebug' }
+                    - { php-version: '5.4', experimental: false, os: ubuntu-latest, coverage-extension: 'xdebug' }
+                    - { php-version: '5.5', experimental: false, os: ubuntu-latest, coverage-extension: 'xdebug' }
+                    - { php-version: '5.6', experimental: false, os: ubuntu-latest, coverage-extension: 'xdebug' }
+                    - { php-version: '8.1', experimental: true, os: ubuntu-latest, coverage-extension: 'pcov' }
```

Comment:
Ignore PHP 8.1 failure somehow that the summary CI icon can be kept green.

Reply:
![image](https://user-images.githubusercontent.com/2228672/113484291-c88c0180-94a7-11eb-8802-31d20f7f2979.png)

if you put `continue-on-error: true` directly to the testing step, there will be no red icon (I think the last step status is displayed, thus maybe one extra Workflow step is needed, or skip the PHP 8.1 error conditionally in bash)